### PR TITLE
[BH-1828] Add labels to relaxation sound list

### PR DESCRIPTION
**Pull request by lukasmas**
Added labels to relaxation

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [x] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-relaxation/widgets/RelaxationListView.hpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+      private:
+        void addItemsOnPage() override;
+        void addLabelMarker(ListItem *item);
+        std::size_t getSlotsLeft();
+
+        MusicType currentType{MusicType::Undefined};
+        MusicType previousType{MusicType::Undefined};
+        MusicType currentMarker{MusicType::Undefined};
+        uint32_t itemsOnPage{0};
```
Comment (dawidwojtas-mudita) [inline products/BellHybrid/apps/application-bell-relaxation/widgets/RelaxationListView.hpp:36]: Use `std::uint32_t`

File: products/BellHybrid/apps/application-bell-onboarding/include/application-bell-onboarding/ApplicationBellOnBoarding.hpp [hunk 2]
```diff
@@ -44,7 +44,7 @@ namespace app
                                            std::string parent                  = "",
                                            StatusIndicators statusIndicators   = StatusIndicators{},
                                            StartInBackground startInBackground = {false},
-                                           std::uint32_t stackDepth            = 1024 * 11);
+                                           std::uint32_t stackDepth            = 1024 * 12);
```
Comment (dawidwojtas-mudita) [inline products/BellHybrid/apps/application-bell-onboarding/include/application-bell-onboarding/ApplicationBellOnBoarding.hpp:47]: Did you check how many bytes we need for this stack?

[the remark this answers is not in the document]
Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-onboarding/include/application-bell-onboarding/ApplicationBellOnBoarding.hpp:47]: Unfortunely. After I introduced a new font to a Linux build it started to crash while onboarding on the simulator.
Maybe we will be able to undo it after we clean up fonts :)

File: products/BellHybrid/apps/application-bell-relaxation/widgets/RelaxationListView.cpp [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            /* Check if new item fits, if it does - add it, if not - handle label insertion
+             * case for bottom-to-top navigation direction. */
+            if (getSlotsLeft() > 0) {
+                body->addWidget(item);
+                itemsOnPage++;
+            }
+            else {
+                break;
+            }
+            /* If direction is bottom-to-top, add label mark after adding phonebook item. */
```
[the remark this answers is not in the document]
Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-relaxation/widgets/RelaxationListView.cpp:54]: Copy - paste mistake :)

File: products/BellHybrid/apps/application-bell-onboarding/include/application-bell-onboarding/ApplicationBellOnBoarding.hpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-onboarding/include/application-bell-onboarding/ApplicationBellOnBoarding.hpp]: Not needed any more.

**Pull request merged**